src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -47,334 +46,13 @@ function writeDrupalDebugPage(files) {
-  );
-
-  // Create the patient and family services page
-  const fsEntityUrl = createEntityUrlObj(drupalPagePath);
-  const fsObj = {
-    careCoordinatorPatientFamilyServices:
-      page.careCoordinatorPatientFamilyServices,
-    socialProgramsPatientFamilyServices:
-      page.socialProgramsPatientFamilyServices,
-    healthWellnessPatientFamilyServices:
-      page.healthWellnessPatientFamilyServices,
-    fieldPatientFamilyServicesIn: page.fieldPatientFamilyServicesIn,
-    facilitySidebar: sidebar,
-    entityUrl: fsEntityUrl,
-    alert: page.alert,
-    title: page.title,
-  };
-  const fsPage = updateEntityUrlObj(
-    fsObj,
-    drupalPagePath,
-    'Patient & Family Services',
-  );
-  files[
-    `drupal${drupalPagePath}/patient-family-services/index.html`
-  ] = createFileObj(
-    fsPage,
-    'health_care_region_patient_family_services_page.drupal.liquid',
-  );
-
-  // Press Release listing page
-  const prEntityUrl = createEntityUrlObj(drupalPagePath);
-  const prObj = {
-    allPressReleaseTeasers: page.allPressReleaseTeasers,
-    facilitySidebar: sidebar,
-    entityUrl: prEntityUrl,
-    title: page.title,
-    alert: page.alert,
-  };
-  const prPage = updateEntityUrlObj(prObj, drupalPagePath, 'Press Releases');
-  paginatePages(
-    prPage,
-    files,
-    'allPressReleaseTeasers',
-    'press_releases_page.drupal.liquid',
-    'press releases',
-  );
-
-  // News Story listing page
-  const nsEntityUrl = createEntityUrlObj(drupalPagePath);
-  const nsObj = {
-    allNewsStoryTeasers: page.allNewsStoryTeasers,
-    fieldIntroTextNewsStories: page.fieldIntroTextNewsStories,
-    facilitySidebar: sidebar,
-    entityUrl: nsEntityUrl,
-    title: page.title,
-    alert: page.alert,
-  };
-  const nsPage = updateEntityUrlObj(
-    nsObj,
-    drupalPagePath,
-    'Community stories',
-    'stories',
-  );
-  paginatePages(
-    nsPage,
-    files,
-    'allNewsStoryTeasers',
-    'news_stories_page.drupal.liquid',
-    'news stories',
-  );
-
-  // Events listing page
-  const eventEntityUrl = createEntityUrlObj(drupalPagePath);
-  const eventObj = Object.assign(
-    { allEventTeasers: page.allEventTeasers },
-    { fieldIntroTextEventsPage: page.fieldIntroTextEventsPage },
-    { facilitySidebar: sidebar },
-    { entityUrl: eventEntityUrl },
-    { title: page.title },
-    { alert: page.alert },
-  );
-  const eventPage = updateEntityUrlObj(eventObj, drupalPagePath, 'Events');
-  paginatePages(
-    eventPage,
-    files,
-    'allEventTeasers',
-    'events_page.drupal.liquid',
-    'events',
-  );
-}
-
 function pipeDrupalPagesIntoMetalsmith(contentData, files) {
   const {
     data: {
       nodeQuery: { entities: pages },
-      sidebarQuery: sidebarNav = {},
-      alerts: alertsItem = {},
-      facilitySidebarQuery: facilitySidebarNav = {},
-      icsFiles: { entities: icsFiles },
     },
   } = contentData;
 
-  const sidebarNavItems = { sidebar: sidebarNav };
```

Comment:
I think these removals are the main place where the code in here is changed. I'm pulling this data out of the full data object in each loop iteration, instead of once at the beginning. This is maybe less efficient, but otherwise I would have to duplicate it in the preview server code.